bot/exts/easter/easter_riddle.py

```diff
@@ -36,6 +36,17 @@ async def riddle(self, ctx: commands.Context) -> None:
         if self.current_channel:
             return await ctx.send(f"A riddle is already being solved in {self.current_channel.mention}!")
 
+        # Don't let users start in a DM
+        if not ctx.guild:
+            await ctx.send(
+                embed=discord.Embed(
+                    title=random.choice(NEGATIVE_REPLIES),
+                    description="You can't start riddles in DMs",
+                    colour=discord.Colour.red()
+                )
+            )
+            return
```

Comment:
This should be above `current_channel` check.

Reply:
I purposely put it below so that if there is a current riddle going on, the user will be notified where.

Reply:
Yeah, alright!